### ISSUE TITLE
Test failures on perl 5.26 because . isn't in @INC

### ISSUE THREAD

**Issue by DrHyde** [robkinyon/dbm-deep#19]
eg ...

```
Can't locate t/common.pm in @INC (you may need to install the t::common module) 
(@INC contains: /home/david/cpantesting/perl-5.26.0/.cpan/build/DBM-Deep-2.0013-
0/blib/lib /home/david/cpantesting/perl-5.26.0/.cpan/build/DBM-Deep-2.0013-0/bli
b/arch /home/david/cpantesting/perl-5.26.0/.cpan/build/DBM-Deep-2.0013-0/_build/
lib /home/david/cpantesting/perl-5.26.0/lib/site_perl/5.26.0/x86_64-linux-thread
-multi /home/david/cpantesting/perl-5.26.0/lib/site_perl/5.26.0 /home/david/cpan
testing/perl-5.26.0/lib/5.26.0/x86_64-linux-thread-multi /home/david/cpantesting
/perl-5.26.0/lib/5.26.0) at t/01_basic.t line 6.
BEGIN failed--compilation aborted at t/01_basic.t line 6.
t/01_basic.t ....................
Dubious, test returned 2 (wstat 512, 0x200)
```

The solution is to `use lib 't'; use common;`.

**Comment by DrHyde** [comment]
See https://github.com/robkinyon/dbm-deep/pull/20 for fix

**Comment by robkinyon** [comment]
Please note I will release this shortly - I'm on a new laptop and I don't have everything installed to release this module - I haven't released in 18 months.

**Comment by robkinyon** [comment]
This has been released as 2.0014

**Comment by DrHyde** [comment]
Thanks!
